config/webpack.config.js

```diff
@@ -53,6 +75,15 @@ module.exports = env => {
     pluginEntries
   );
 
+  var publicPath;
+  if (production) {
+    publicPath = process.env.ASSET_PATH || '/webpack/';
+  } else {
+    publicPath = process.env.ASSET_PATH || `${devServerProtocol}://${devServerBindHost}:${devServerPort}/webpack/`;
+  }
+
+  const supportedLanguagesRE = supportedLanguages().join('|');
```

Comment:
That makes sense. I'll change it.

Reply:
While you're at it, it looks like you missed the `^` at the beginning.

Reply:
No, I intentionally match only the end of the path to be able to re-use the same pattern for both intl and react-intl.

Reply:
I wonder if you could have any issues with for example `een` instead of `en` because of this, but I think that's unlikely because it doesn't really handle a lot of user input and reuse is probably worth more.